creator-node/src/services/sync/secondarySyncFromPrimary.js

```diff
@@ -196,6 +196,13 @@ module.exports = async function (
         clock: fetchedLatestClockVal,
         clockRecords: fetchedClockRecords
       } = fetchedCNodeUser
+      logger.info(
+        logPrefix,
+        `fetchedCNodeUser ${JSON.stringify(fetchedCNodeUser)}`
```

Comment:
we can actually delete this log, we don't need to print this out everytime. was for debugging